admin_pages/payments/Payment_Log_Admin_List_Table.class.php

```diff
@@ -198,8 +198,10 @@ public function column_PMD_ID(EE_Change_Log $item)
             return $item->object()->admin_name();
         } elseif ($item->object() instanceof EE_Payment && $item->object()->payment_method()) {
             return $item->object()->payment_method()->admin_name();
+        } elseif ($item->object() instanceof EE_Transaction) {
+            return esc_html__('Unknown', 'event_espresso');
```

Comment:
Why using "Unknown" here?  Is there some other label that can be used that might provide some more helpful information?  Or would it be better to maybe exclude logs that are specifically a transaction object from showing in this list table instead?

Or maybe it'd be better to get the payment method from the transaction and then use _it's_ `admin_name()`?

Reply:
Good question and the reason for unknown is we can't be certain that this log is for the transactions last payment method which makes it harder for support to follow whats happening and can easily lead us down the wrong path.

Lets say a user attempts a payment with payment method A and it shows as payment failed, so switches to payment method B and it's successful, both logs will then show as payment method B.

Mike initially did use the payment method name from the transaction and I changed it because of the above, see:
https://events.codebasehq.com/projects/event-espresso/tickets/11367#update-49090081